frontend/src/app/oe/components/base-box-hor/base-box-hor.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, OnInit, OnDestroy, ChangeDetectionStrategy, Input } from '@angular/core';
+import { StateService } from 'src/app/services/state.service';
+import { ActivatedRoute, ParamMap } from '@angular/router';
+import { RelativeUrlPipe } from 'src/app/shared/pipes/relative-url/relative-url.pipe';
+
+export const MAX_COUNT = 14;
+@Component({
+  selector: 'app-base-box-hor',
+  templateUrl: './base-box-hor.component.html',
+  styleUrls: ['./base-box-hor.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BaseBoxHorComponent implements OnInit, OnDestroy {
+  @Input() type: 'Energy' | 'Strike' | 'Strike_Boiling' = 'Energy';
+  @Input() color = 'red';
```

Comment:
I see in most of the components we have red as defalt can you please move this to constant and use it if possible?